code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -817,3 +817,19 @@
 
 /datum/reagent/xeno_neurotoxin/overdose_crit_process(mob/living/M)
 		M.adjustOxyLoss(min(4,volume * 0.2 * REM)) //Overdose starts applying more oxy damage
+
+/datum/reagent/xeno_growthtoxin
```

Comment:
shouldnt this be in the other PR not this one?

Reply:
No, because it's both integral and exclusive to the Defiler